src/components/Views/SyncView/SyncView.js

```diff
@@ -155,6 +156,10 @@ class SyncView extends React.Component<
 
     const keyExtractor = (item, index) => item.id;
 
+    const handleUSBPress = () => {
+      startSync({filename: '/sdcard/Android/data/com.mapeomobile/static/data.syncfile'})
```

Comment:
Is this path consistent for all Android phones?

Reply:
no :P we definitely want to change this